### PR TITLE
Add daily security scan for NFS Provisioner image

### PR DESCRIPTION
**Pull request by roni-frantchi**
In #14 we've added an NFS provisioner.  
We would like to get daily notifications on its security vulnerabilities

### CODE REVIEW

**Comment by roni-frantchi** [comment]
[Keep some security scan of google's nfs server docker image](https://trello.com/c/b8W6hXR9/5333-keep-some-security-scan-of-googles-nfs-server-docker-image)

File: .github/workflows/docker-scan.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+name: Docker Image Scan
+
+on:
+  schedule:
+    - cron:  '0 9 * * *'
+
+env:
+  SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK_URL }}
+  SLACK_CHANNEL: dev
```
Comment (GiliFaroEnv0) [inline .github/workflows/docker-scan.yml:9]: why not `#epic-k8s-agent`?

**Review by GiliFaroEnv0 — APPROVED** [review]
LGTM!
The only question is why not the relevant channel?

File: .github/workflows/docker-scan.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  SLACK_USERNAME: e2eBot
```
Comment (avnerenv0) [inline .github/workflows/docker-scan.yml]:
```suggestion
  SLACK_USERNAME: somethingElseBot
```

**Review by avnerenv0 — APPROVED** [review]
LGTM but for future generations maybe we also want to specify that curv/paypal should be notified?

File: .github/workflows/docker-scan.yml (hunk 1 above)
Comment (roni-frantchi) [reply .github/workflows/docker-scan.yml:9]: That would work as well.  
Just thought there's more to it than an epic reference (like letting customers know etc.)

File: .github/workflows/docker-scan.yml (hunk 2 above)
Comment (roni-frantchi) [reply .github/workflows/docker-scan.yml]:
```suggestion
  SLACK_USERNAME: securityBot
```

Comment (roni-frantchi) [reply .github/workflows/docker-scan.yml]: [`8f2ac80` (#15)](https://github.com/env0/k8s-modules/pull/15/commits/8f2ac800317342d5d44c9f8c28d3e93bc6c7b1e1)

**Pull request merged**